### PR TITLE
docs: update product-alerts.component.1.ts

### PR DESCRIPTION
**Pull request by bradlygreen**
Make the example match the generated output seen in StackBlitz as shown https://next.angular.io/getting-started#input step 2 "Open product-alerts.component.ts."

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by IgorMinar — APPROVED** [review]

File: aio/content/examples/getting-started/src/app/product-alerts/product-alerts.component.1.ts [hunk 1]
```diff
@@ -12,8 +12,14 @@ import { Input } from '@angular/core';
   styleUrls: ['./product-alerts.component.css']
 })
 // #docregion input-decorator
-export class ProductAlertsComponent {
+export class ProductAlertsComponent implements OnInit {
 // #enddocregion as-generated
   @Input() product;
 // #docregion as-generated
+
```
Comment (IgorMinar) [inline aio/content/examples/getting-started/src/app/product-alerts/product-alerts.component.1.ts]:
```suggestion

```

**Review by IgorMinar — COMMENTED** [review]
CI fails because of the trailing whitespace, please fix.

**Comment by benlesh** [comment]
@bradlygreen Looks like you just need to squash your commits and this is good to go.

**Comment by benlesh** [comment]
Caretaker note: test_aio CI was failing because this was an edit via the GitHub UI... merging.

**Pull request closed without merging**